README.md

```diff
@@ -27,17 +27,37 @@
 > This action presumes you already have a zeroheight account. You can [sign up here](https://zeroheight.com/create/account?utm_department=marketing&utm_source=github).
 
 1. Head to [`zeroheight.com/adoption`](https://zeroheight.com/adoption) and follow the steps to connect your design system packages and application source code to zeroheight
-3. Use the component tracking setup steps to create a Client ID and Access Token and ensure to note these down for future use
+3. Follow the "Use the CLI" flow to create a Client ID and Access Token and ensure to note these down for future use
 4. Now head over to your application's source code on GitHub and enter your zeroheight Client ID and Access Token as GitHub repository secrets (Settings → Secrets and variables → Actions) using `ZEROHEIGHT_CLIENT_ID` and `ZEROHEIGHT_ACCESS_TOKEN` respectively
-5. Once complete, add the following step to your existing GitHub Action workflow file:
-   ```yaml
-      - name: Analyze design system adoption data
-        uses: zh-ski/action-design-system-adoption@v1
-        env:
-          ZEROHEIGHT_CLIENT_ID: "${{ secrets.ZEROHEIGHT_CLIENT_ID }}"
-          ZEROHEIGHT_ACCESS_TOKEN: "${{ secrets.ZEROHEIGHT_ACCESS_TOKEN }}"
-   ```
-   We recommend adding it as part of your post-deployment workflows. If you're new to workflows and actions check out [GitHub's Actions documentation](https://docs.github.com/en/actions) for more info.
-6. Save your changes, trigger your action, and check in zeroheight to see design system usage data come in 🎉
+5. Once completed you can use the action as part of your GitHub Action workflow file
+
+## Usage
+
+```yaml
+  - name: Analyze design system adoption data
+    uses: zh-ski/action-design-system-adoption@v2
+    with:
+      command: 'analyze' # or track-package, monitor-repo
```

Comment:
`with` and `command` lines can be removed if we provide a default command value in the action itself